app/Listeners/Discord/SendSpeedtestCompletedNotification.php

```diff
@@ -42,6 +42,8 @@ public function handle(SpeedtestCompleted $event): void
                 'ping' => round($event->result->ping).' ms',
                 'download' => Number::toBitRate(bits: $event->result->download_bits, precision: 2),
                 'upload' => Number::toBitRate(bits: $event->result->upload_bits, precision: 2),
+                'downloaded_bytes' => Number::bytesToHuman(bytes: $event->result->downloaded_bytes, precision: 2),
```

Comment:
We're in the process of rewriting the notification channels, so we're not moving the goal post can you please remove anywhere you added the `_bytes` to notification channels?